editor/src/clj/editor/sprite.clj

```diff
@@ -393,16 +394,8 @@
       (g/set-property self :material material)
       (g/set-property self :blend-mode (:blend-mode sprite))
       (g/set-property self :size-mode (:size-mode sprite))
-      (g/set-property self :size (v4->v3 (:size sprite)))
+      (g/set-property self :manual-size (v4->v3 (:size sprite)))
       (g/set-property self :slice9 (:slice9 sprite))
-
-      ;; The size property value clause is dependent on size metadata from the
-      ;; image, and the old value is evaluated whenever the property is set (for
-      ;; use in the property setter function). Assigning the image property
-      ;; after the size property ensures the dependent property value clause
-      ;; does not perform the expensive image size metadata computation for the
-      ;; ignored old-value of the size property. This saves immensely on load
-      ;; times for projects with a lot of sprites.
```

Comment:
Yay, this change gets rid of the problem explained by this apologetic comment! 🥳